README.md

```diff
@@ -1,25 +1,30 @@
-[![Godot Engine logo](/logo.png)](https://godotengine.org)
+# Godot Engine
 
-## Godot Engine
+<p align="center">
+  <a href="https://godotengine.org">
+    <img src="logo.svg" width="400" alt="Godot Engine logo">
+  </a>
+</p>
 
-Homepage: https://godotengine.org
+## 2D and 3D cross-platform game engine
 
-#### 2D and 3D cross-platform game engine
+**[Godot Engine](https://godotengine.org) is a feature-packed, cross-platform
+game engine that can be used to create 2D and 3D games from a unified
```

Comment:
The addition of "that can be used" makes it a bit wordy IMO. I know @pycbouh raised the concern that the verb seemed missing, but AFAIK it wasn't incorrect before: "a hammer is a tool to put nails in surfaces" / "Godot is a game engine to create games".

Reply:
I didn't want to imply that it was incorrect, just that it didn't flow well when I was reading it. I think that additional qualifiers after "to create" make it much weirder to read than "a tool to put nails".

Reply:
I agree that the sentence is a bit heavy on qualifiers which make it a bit wordy. I wasn't convinced by adding "that can be used", but I'm open to other suggestions to improve this sentence so that it reads better.